pkg/model/names.go

```diff
@@ -136,6 +138,8 @@ func (b *KopsModelContext) IAMName(role kops.InstanceGroupRole) string {
 	switch role {
 	case kops.InstanceGroupRoleMaster:
 		return "masters." + b.ClusterName()
+	case kops.InstanceGroupRoleAPIServer:
+		return "apiserver." + b.ClusterName()
```

Comment:
Should that be "apiservers" (pluralized) to align with the rest of the role names here?